tests/functional/spec/census/issues/routing-last-week-were-you.spec.js

```diff
@@ -88,4 +89,98 @@ describe('ArriveInUk', function () {
+    startCensusQuestionnaire('census_household.json')
+
+    // who-lives-here
+    PermanentOrFamilyHome.clickPermanentOrFamilyHomeAnswerYes().submit()
+    HouseholdComposition.setFirstName('John').addPerson().setFirstName('Jane', 1).submit()
+    EveryoneAtAddressConfirmation.clickEveryoneAtAddressConfirmationAnswerYes().submit()
+    OvernightVisitors.setOvernightVisitorsAnswer(0).submit()
+    HouseholdRelationships.clickHouseholdRelationshipsAnswerHusbandOrWife().submit()
+    WhoLivesHereCompleted.submit()
+
+    // household-and-accommodation
+    TypeOfAccommodation.clickTypeOfAccommodationAnswerWholeHouseOrBungalow().submit()
+    TypeOfHouse.clickTypeOfHouseAnswerSemiDetached().submit()
+    SelfContainedAccommodation.clickSelfContainedAccommodationAnswerYesAllTheRoomsAreBehindADoorThatOnlyThisHouseholdCanUse().submit()
+    NumberOfBedrooms.setNumberOfBedroomsAnswer(3).submit()
+    CentralHeating.clickCentralHeatingAnswerGas().submit()
+    OwnOrRent.clickOwnOrRentAnswerOwnsOutright().submit()
+    NumberOfVehicles.setNumberOfVehiclesAnswer(2).submit()
+    HouseholdAndAccommodationCompleted.submit()
+
+    // household-member
+    HouseholdMemberBegin.submit()
+    DetailsCorrect.clickDetailsCorrectAnswerYesThisIsMyFullName().submit()
+    Over16.clickOver16AnswerYes().submit()
+    PrivateResponse.clickPrivateResponseAnswerNoIDoNotWantToRequestAPersonalForm().submit()
+    Sex.clickSexAnswerMale().submit()
+    DateOfBirth.setDateOfBirthAnswerDay(2).setDateOfBirthAnswerMonth(3).setDateOfBirthAnswerYear(1980).submit()
+    MaritalStatus.clickMaritalStatusAnswerMarried().submit()
+    AnotherAddress.clickAnotherAddressAnswerNo().submit()
+    InEducation.clickInEducationAnswerNo().submit()
+    CountryOfBirth.clickCountryOfBirthEnglandAnswerEngland().submit()
+    Carer.clickCarerAnswerNo().submit()
+    NationalIdentity.clickNationalIdentityEnglandAnswerEnglish().submit()
+    EthnicGroup.clickEthnicGroupEnglandAnswerWhite().submit()
+    WhiteEthnicGroup.clickWhiteEthnicGroupEnglandAnswerEnglishWelshScottishNorthernIrishBritish().submit()
+    Language.clickLanguageEnglandAnswerEnglish().submit()
+    Religion.clickReligionAnswerNoReligion().submit()
+    PastUsualAddress.clickPastUsualAddressAnswerThisAddress().submit()
+    Passports.clickPassportsAnswerUnitedKingdom().submit()
+    Disability.clickDisabilityAnswerNo().submit()
+    Qualifications.clickQualificationsEnglandAnswerUndergraduateDegree().submit()
+    Volunteering.clickVolunteeringAnswerNo().submit()
+    EmploymentType.clickEmploymentTypeAnswerWorkingAsAnEmployee().clickEmploymentTypeAnswerOnAGovernmentSponsoredTrainingScheme().clickEmploymentTypeAnswerNoneOfTheAbove().submit()
+    expect(Jobseeker.isOpen()).to.be.false
+  })
+
+  it('Given I am answering question 28.	Last week were you.., When I select multiple answers excluding none of the above, Then I am routed to 35. In your main job, are (were) you?', function () {
```

Comment:
This test description doesn't match up to the test - it doesn't check that you are routed to question 35, it checks that you are not routed to question 29.